.gitignore

```diff
@@ -13,3 +13,5 @@ jsdoc
 deployment-config.json
 *~
 #*
+*.in
```

Comment:
that seems wrong. we do want the *.in files. We should only ignore system generate files, temp files, build products, etc. I guess what I'm saying is ignore anything that is _not_ a source file or related.